src/components/reactive-search-wrappers/RSMultiList.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { MultiList } from '@appbaseio/reactivesearch';
+import PropTypes from 'prop-types';
+
+// Css class name helper
+const multiListInnerClass = {
+  title: 'rs-facet-title',
+  list: 'rs-facet-list',
+  label: 'rs-facet-label'
+};
+
+const RSMultiList = props => {
+  const { allFilters, defaultVal, facet, title } = props;
+
+  return (
+    <MultiList
+      componentId={facet.name.replace(/\s+/g, '')}
+      dataField={facet.field}
+      defaultSelected={defaultVal}
+      innerClass={multiListInnerClass}
+      missingLabel="None"
+      react={{
+        and: allFilters.filter(entry => {
+          return entry !== facet.name.replace(/\s+/g, '');
+        })
+      }}
+      showCheckbox={false}
+      showMissing={true}
+      showSearch={false}
+      title={title}
```

Comment:
```suggestion
     title={facet.name}
```

Reply:
So I'm purposely passing in `title` as an extra prop to this component, because there's one use case where I want the prop to be there, but have an intended empty string value.  On the collections item page, the default title doubles up the "Date" Collapsible header.   I know it looks weird, but that was my reasoning.